server/asset/btc/testing.go

```diff
@@ -317,69 +318,37 @@ out:
 	} else {
 		t.Logf("no unknown script types")
 	}
+	// Fees
+	feeCount := len(stats.feeRates)
+	if feeCount > 0 {
+		var feeSum uint64
+		for _, r := range stats.feeRates {
+			feeSum += r
+		}
+		t.Logf("%d fees, avg rate %d", feeCount, feeSum/uint64(feeCount))
+	}
```

Comment:
For the commit message on merge, can you comment in the PR on the fee-related changes too?